01_Cheminfo_crash_course.ipynb

```diff
@@ -232,7 +232,7 @@
         "# 3.\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    # 6. CC(C(=O)N)CC
</pre><p> </p><p>This change was intentional to give crash course participants something to work with.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MolSSI-Education/iqb-2025/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>